pkg/sdn/plugin/subnets.go

```diff
@@ -155,6 +165,37 @@ func (master *OsdnMaster) watchNodes() {
 	}
 }
 
+// Because openshift-sdn uses an overlay and doesn't need GCE Routes,
+// we need to clear the kubelet-added NetworkUnavailable condition
+// that kubelet adds to initial node status when using GCE.
+// TODO: make upstream kubelet more flexible with overlays and GCE
+func (node *OsdnNode) clearInitialNodeNetworkingCondition() error {
+	knode, err := node.registry.GetNode(node.hostName)
+	if err != nil {
+		return err
+	}
+
+	for i := 0; i < updateNodeStatusMaxRetries; i++ {
```

Comment:
Retry in this case will always fail.
We need to fetch the latest node object before retrying (otherwise resourceversion will not match). Check tryUpdateNodeStatus() in pkg/kubelet/kubelet.go